.github/workflows/docs.yml

```
@@ -56,10 +59,7 @@ jobs:
           echo "<meta name=\"robots\" content=\"noindex\">" > header.html
 
       - name: Build docs
-        env:
-          # needs to be in sync with [package.metadata.docs.rs]
-          RUSTDOCFLAGS:  -Zunstable-options --cfg=docsrs
-        run: cargo doc --all-features --no-deps -p bevy -Zunstable-options -Zrustdoc-scrape-examples
+        run: cargo docs-rs -p bevy --version ^0.1.12
```

```suggestion
        run: cargo docs-rs -p bevy
```
This needs to be on the install command as per @BD103 comment, not here